javascript/javascript1/week3/phone-usage.js

```diff
@@ -0,0 +1,38 @@
+const activities = [];
+let date = new Date();
+let today = date.toLocaleDateString();
+
+function addActivity(date, activity, period) {
+    activities.push({ date, activity, period });
+}
+// adding acctivities to the empty array
+addActivity(today, "YouTube", 30);
+addActivity(today, "Facebook", 34);
+addActivity(today, "Twitter", 14);
+//addActivity(today, "Instagram", 12); // no smartphoning alert pops up when it reachs the limit
+
+console.log(activities);
+
+function showStatus(data) {
+    let totalUsage = 0;
+    const maxLimit = 90; // in minutes
+    for (let i = 0; i < data.length; ++i) {
+        totalUsage += activities[i].period;
+    }
+    console.log(
+        "You have added " +
+        activities.length +
+        " activities. They amount to " +
+        totalUsage +
+        " min. of usage"
+    );
+    if (totalUsage >= maxLimit) {
+        console.log("You have reached your limit, no more smartphoning for you!");
+    } else if (activities == "" || activities == null) {
+        console.log("Add some activities before calling showStatus");
+    }
```

Comment:
activities is an array and here you compare it with an empty string. As I told you up, check this array length and remove this